lib/AST/ASTImporter.cpp

```diff
@@ -1604,12 +1660,12 @@ Decl *ASTNodeImporter::VisitAccessSpecDecl(AccessSpecDecl *D) {
   if (!DC)
     return nullptr;
 
-  AccessSpecDecl *accessSpecDecl
-    = AccessSpecDecl::Create(Importer.getToContext(), D->getAccess(),
-                             DC, Loc, ColonLoc);
-
-  if (!accessSpecDecl)
-    return nullptr;
+  AccessSpecDecl *accessSpecDecl;
```

Comment:
Maybe if you already touching this part you could also fix the naming and have uppercase initial for the local variable.